### DISCUSSION TITLE
Looking for the blank HFS collection macHD.zip

### DISCUSSION THREAD

**Discussion by jamieson-olsen** [erichelgeson/BlueSCSI-v1#102] [Q&A]
There used to be a nice collection of HFS formatted images here (macHD.zip) but that seems to have moved. There is a link to the mega.nz site but that archive has only a few size options. Anyone have the link to the macHD.zip file?

**Comment by erichelgeson** [comment]
Those were removed as they contained Lido drivers which are extremely slow. You can try DiskJockey https://github.com/OneGeekArmy/Disk-Jockey or I could add in a custom size for you to the zip.

**Reply by erichelgeson** [reply]
I've added a new zip with tons of pre-formatted sizes, hopefully will meet your needs.

**Reply by jamieson-olsen** [reply]
I also tried making images with "dd" but of course they didn't contain the SCSI driver software component (even after the Baskilisk-II emulator initialized them). The presence of a SCSI driver must be the difference between a "volume" and a "drive image". I'll give your new drive images a shot, 2GB is the size I was looking for. Thanks for making this great project!

**Reply by erichelgeson** [reply]
Right, after dd it'll be nothing, just like a blank unformatted/unpartitioned hdd - you'd need to boot up a real mac and format/partition it. Baskilisk II doesnt know anything about SCSI, it just finds thinks that look like HFS volumes and mounts them.

**Reply by erichelgeson** [reply]
FYI I just added a 2048mb (2gb) blank to that archive. Let me know if it works for you!

**Reply by Elv1sP** [reply]
I need a 20gb HFS+ volume for my power macs, can you fix? Read that hfs is slow and bluescsi supports hfs+ volumes that should work better and faster in ppc macs, true?

**Reply by erichelgeson** [reply]
Please use Disk Jockey (see Usage page) to create whatever drive size you like (you'll need to enable large disks in Disk Jockey's settings) . Format it with any OS utility to whatever file system you like, BlueSCSI knows nothing of the file system on the image.

**Reply by necronomfive** [reply]
DiskJockey is severely limited on Windows. Is there an alternative to create such partitions fow Windows users?

**Reply by erichelgeson** [reply]
Use the same approach as mentioned above. Create a blank (or not blank) file and use it as an image. Then format it with utilities on the retro machine you're using BlueSCSI in.

**Comment by jamieson-olsen** [comment]
I see the new images uploaded today and I've tried the 2048MB, 1600MB and 200MB versions. I use Baskilisk-II to boot the Apple Recovery CD, then I do a fresh install of 7.5.3 onto the blank drive image, then upgrade to 7.5.5. and then copy the drive image file to the MicroSD card. The BlueSCSI log file reports that it finds the drive image OK, yet my SE/30 refuses to load it. If I repeat the process with a different blank drive image (don't recall where I got it) it boots fine on the SE/30. Could there be some compatibilty issue with the SCSI driver used on the blank images you provided?

**Reply by erichelgeson** [reply]
The more we know about Mac's SCSI drivers the more confusing it is. I'll take a look. 

This is git so nothing is ever really gone - https://github.com/erichelgeson/BlueSCSI/blob/3b17788cc677b7b089875786be94f929d4913bce/docs/MacHD.zip - but please open an apple drive utility from your install and click update driver so you're not using Lido.

**Comment by jamieson-olsen** [comment]
Can confirm that I take one of the blanks from the latest ZIP file and load it on the BlueSCSI. My SE/30 cannot see it. Then on the SE/30, I run the Apple HD SC Setup 7.3.5 utility, select that SCSI device, update driver, and reboot. Then the new drive shows up. So it seems that whatever SCSI driver was used to make those blanks is not compatible with the SE/30 hardware. Thanks for your help on this.
